.travis/run.sh

```diff
@@ -85,6 +85,60 @@ for BOARD in $BOARDS; do
 				PROG=$PROG BOARD=$BOARD TARGET=$TARGET make load || true
 			done
 		fi
+		
+		# Copy built files
```

Comment:
We don't want to run this for pull requests (as they don't have access to the full Xilinx tools). Maybe we should be looking at something like http://docs.travis-ci.com/user/environment-variables/#Convenience-Variables ?